docs/content/guide/decorators.ngdoc

```diff
@@ -0,0 +1,356 @@
+@ngdoc overview
+@name Decorators
+@sortOrder 345
+@description
+
+# Decorators in AngularJS
+
+<div class="alert alert-warning">
+  **NOTE:** This guide is targeted towards developers who are already familiar with AngularJS basics.
+  If you're just getting started, we recommend the {@link tutorial/ tutorial} first.
+</div>
+
+## What are decorators?
+
+Decorators are used to facilitate a decorator design pattern. This pattern is used to separate modification or
+*decoration* of a class without modifying the original source code. In Angular, decorators are functions that allow a
+service to be modified after their instantiation.
```

Comment:
1. It's not about `services`. Filters and directives can be decorated as well.
2. Not after their instantiation. Did you mean after their declaration/registration/definition or something ?